yang/ietf-network-inventory.yang

```diff
@@ -316,7 +324,8 @@ module ietf-network-inventory {
           description
             "The NE type.";
         }
-        uses common-entity-attributes;
+        uses minimal-common-entity-attributes;
+        uses ne-component-common-entity-attributes;
```

Comment:
See previous comment (defining the ne-component-common-entity-attributes as a super-set of the common attributes which includes also the basic common attributes)

```suggestion
        uses ne-component-common-entity-attributes;
```